src/pages/iou/request/MoneyRequestAttendeeSelector.tsx

```diff
@@ -136,17 +150,22 @@ function MoneyRequestAttendeeSelector({attendees = [], onFinish, onAttendeesAdde
      * Returns the sections needed for the OptionsSelector
      */
     const [sections, header] = useMemo(() => {
-        const newSections: OptionsListUtils.Section[] = [];
+        const newSections: Array<SectionListDataType<Option>> = [];
         if (!areOptionsInitialized || !didScreenTransitionEnd) {
             return [newSections, ''];
         }
         const fiveRecents = [...chatOptions.recentReports].slice(0, 5);
         const restOfRecents = [...chatOptions.recentReports].slice(5);
         const contactsWithRestOfRecents = [...restOfRecents, ...chatOptions.personalDetails];
 
-        const formatResults = OptionsListUtils.formatSectionsFromSearchTerm(
+        const formatResults = formatSectionsFromSearchTerm(
             debouncedSearchTerm,
-            attendees.map((attendee) => ({...attendee, reportID: attendee.reportID ?? '-1'})),
+            attendees.map((attendee) => ({
+                ...attendee,
+                reportID: attendee.reportID ?? CONST.DEFAULT_NUMBER_ID.toString(),
```

Comment:
NAB - Does `attendee` ever have a `reportID` here, or can we just directly do `CONST.DEFAULT_NUMBER_ID.toString()` instead?

Reply:
It does not.